src/main/java/com/google/sps/placeGuide/PlaceGuide.java

```diff
@@ -0,0 +1,162 @@
+package com.google.sps.placeGuide;
+
+import org.jetbrains.annotations.Nullable;
+import com.google.appengine.api.datastore.GeoPt;
+
+/** Class containing place guide's information. */
+public class PlaceGuide {
+
+  // Unique identifier for a {@code PlaceGuide} automatically given 
+  // when a {@code PlaceGuide} entity is created and will be used to delete
+  // or edit {@code PlaceGuide}.
+  @Nullable
+  private final long id;
+
+  private final String name;
+  private final String audioKey; 
+  private final String creatorId;
+  private final boolean isPublic;
+
+  // This is not the unique identifier of a place guide.
```

Comment:
then what is this?

Reply:
It is the unique identifier of the place based on the Google Maps Places API. It will null if it's not from Google Maps Places API. I have edited the javadoc.